src/main/scripts/setup/hydra_setup_script.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
Can we have only one launcher script ? (Similar to aurora)
you can call other functions internally from this script and the help on this script can describe all the other scripts.